api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+          (e.g., using HTML).
+          This Problem object is the same as provided by https://github.com/zalando/problem
+        example: http://httpstatus.es/503
+      title:
+        type: string
+        description: |
+          A short, summary of the problem type. Written in english and readable
+          for engineers (usually not suited for non technical stakeholders and
+          not localized); example: Service Unavailable
+      status:
+        type: integer
+        format: int32
+        description: |
+          The HTTP status code generated by the origin server for this occurrence
+          of the problem.
+        example: 503
+      detail:
+        type: string
+        description: |
+          A human readable explanation specific to this occurrence of the
+          problem.
+        example: Connection to database timed out
+      instance:
+        type: string
+        format: uri
+        description: |
+          An absolute URI that identifies the specific occurrence of the problem.
+          It may or may not yield further information if dereferenced.
+    required:
+      - type
+      - title
+      - status
+
+  Metrics:
+    type: object
+
+  Topic:
+    description: |
+      Representation of the underlying (implementation dependent) event ordering and persistence structure.
+
+      Direct operation on this is usually unnecessary.
+    type: object
+    required:
+      - name
+    properties:
+      name:
+        type: string
+        description: Topic name
+        example: article.120
+
+  Partition:
+    required:
+      - partition
+      - oldest_available_offset
+      - newest_available_offset
+    properties:
+      partition:
+        type: string
+      oldest_available_offset:
+        type: string
+      newest_available_offset:
+        type: string
+
+  Cursor:
+    required:
+      - partition
+      - offset
+    properties:
+      partition:
+        type: string
+      offset:
+        type: string
+
+  EventStreamBatch:
+    description: |
+      One chunk of events in a stream. A batch consists of an array of `Event`s plus a `Cursor` pointing to
+      the offset of the last Event in the stream.
+
+      The size of the array of Event is limited by the parameters used to initialize a Stream.
+    required:
+      - cursor
+    properties:
+      cursor:
+        $ref: '#/definitions/Cursor'
+      events:
+        type: array
+        items:
+          $ref: '#/definitions/Event'
+
+  EventType:
+    description: An event type defines the schema and its runtime properties.
+    properties:
+      name:
+        type: string
+        description: |
+          Name of this EventType. Encodes the owner/responsible for this EventType.
+
+          The name for the EventType SHOULD follow the pattern, but is not enforced
+          'organization:team:owning_application:event-type', for example
+          'zalando:bazaar:price-service:price-change'.
```

Comment:
Does anything say against using `.` as a separator for the scopes? (I have already commented on that higher)